src/gotoDefinition.ts

```diff
@@ -48,4 +53,9 @@ export const registerGotoDefinition = () => {
             return [{ range, targetRange: range, uri, targetUri: uri }]
         },
     })
+    setFindReferenceButtonVisibility()
+
+    vscode.workspace.onDidChangeConfiguration(({ affectsConfiguration }) => {
+        if (affectsConfiguration('veturExtended.enableFindReferencesButton')) setFindReferenceButtonVisibility()
```

Comment:
With [vscode-framework](https://github.com/zardoy/vscode-framework) I try to avoid using extension id or name in its source code. With this in mind other folks can easily rename their extension or publish with different id with different setting ids.

So, we have the following wrappers from vscode-framework:

- `getExtensionSetting` to get typed value of the setting
- `getExtensionSettingId` to get full id of the setting (should be used in this place)

Also, they receieve typed argument, so in case if we rename the setting and don't rename it here for some reason we would receive an error.